mozilla_vpn/views/exchange_rates.view.lkml

```diff
@@ -0,0 +1,41 @@
+include: "//looker-hub/mozilla_vpn/views/exchange_rates_table.view"
```

Comment:
this view file is unnecessary, as it doesn't change any of the values for `exchange_rates_table`

Reply:
Thanks!  I have removed the view.